x/incentives/types/msgs.go

```diff
@@ -55,10 +55,6 @@ func (m MsgCreateGauge) ValidateBasic() error {
 		return errors.New("distribution period should be 1 epoch for perpetual gauge")
 	}
 
-	if lockuptypes.LockQueryType_name[int32(m.DistributeTo.LockQueryType)] != "ByDuration" {
```

Comment:
why was this removed as well?

Reply:
In CreateGauge for CL pool we don't want to limit just using `ByDuration` right? Or lmk if im wrong